### PR TITLE
Add lisk console to application command - Closes #6306

### LINKED ISSUE #6306: Add lisk console to application command
### Description

Add a command to the lisk-commander application commands to create repl environment with an initialized lisk-client library.

### Motivation

Lisk SDK developers do not have the ability to execute functions against running blockchain instances or use library utility or functions to perform offline operations to create a transaction from console.

### Specification

The following 3 patterns should be supported:

```
lisk-app console
lisk-app console --api-ws=ws://xxxx
lisk-app console --api-ipc=datapath
```

If api is not specified, all `lisk-client` libraries should be imported at the root level. For example, `@liskhq/lisk-transactions` should be accessible by `transactions`.

If `api-ws` or `api-ipc` is specified, variable `client` should be initialized with connected API client.

### Acceptance Criteria

- 3 patterns are working against devnet node
- all 3 patterns are tested in the unit test
- check REPL contains the imported library name space

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #6306 

### How was it solved?

- Introduced new console command

### How was it tested?

- Added unit tests

For manual testing
- Initialise console with lisk client only `./commander/bin/run console`
- Initialise console with lisk client and ws client `./commander/bin/run console --api-ws=ws://localhost:8080/ws`
- Initialise console with lisk client and ipc client `./commander/bin/run console --api-ipc=/path/to/client`

To use await in console
- `NODE_OPTIONS=--experimental-repl-await ./commander/bin/run console --api-ws=ws://localhost:8080/ws`

### CODE REVIEW

File: commander/src/bootstrapping/commands/console.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		}
+
+		if (flags['api-ipc']) {
+			const ipcClient = await liskClient.apiClient.createIPCClient(flags['api-ipc']);
+			replServer.context.client = ipcClient;
+		}
+
+		if (flags['api-ws']) {
+			const wsClient = await liskClient.apiClient.createWSClient(flags['api-ws']);
+			replServer.context.client = wsClient;
```
Comment (nazarhussain) [inline commander/src/bootstrapping/commands/console.ts:58]: We should create these context objects readonly. Please refer to the docs they have an example there.

File: commander/src/bootstrapping/commands/console.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		}
```
Comment (nazarhussain) [inline commander/src/bootstrapping/commands/console.ts:59]: All the user define context can be reset if user enter `. clear` command. So we should listen to `reset` event and initialize the context again.

File: commander/src/bootstrapping/commands/console.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+			exclusive: ['api-ipc'],
+		}),
+	};
+
+	async run(): Promise<void> {
+		const { flags } = this.parse(ConsoleCommand);
+
+		this.log('Entering Lisk REPL: type `Ctrl+C` or `.exit` to exit');
+
+		const replServer = repl.start('lisk > ');
```
Comment (nazarhussain) [inline commander/src/bootstrapping/commands/console.ts:45]: I am not sure if we always use `lisk >` or the user specific app label.

**Review by nazarhussain — CHANGES_REQUESTED** [review]

File: commander/src/bootstrapping/commands/console.ts (hunk 3 above)
Comment (shuse2) [reply commander/src/bootstrapping/commands/console.ts:45]: I think we should use the application name

**Review by shuse2 — COMMENTED** [review]
Other than @nazarhussain 's comment LGTM

**Review by shuse2 — APPROVED** [review]

**Review by nazarhussain — APPROVED** [review]

**Pull request merged**